.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["@react-bootstrap/babel-preset"]
+  "presets": ["@react-bootstrap", "@babel/typescript"]
```

Comment:
How can it be `@react-bootstrap`?

Reply:
It's a shorthand Babel understands, eslint work similarly as well. Essentially just appends 'babel-preset' to the end when it sees a lone scope